### ISSUE TITLE
Abstract not up to date

### ISSUE THREAD

**Issue by MartinMcGrath** [mjemmeson/Date-Holidays-GB#5]
Hi, the Abstract reads: "Determine British holidays - Current UK public and bank holiday dates up to 2019", while the .013 release includes dates up to the end of 2021

**Comment by mjemmeson** [comment]
Thanks! Fixed in v0.014

**Comment by MartinMcGrath** [comment]
It seems that metacpan still finds this 

![image](https://user-images.githubusercontent.com/1374568/74655867-01bec080-5185-11ea-93cf-19214a815a61.png)


```
grep -r "up to 2019" *
lib/Date/Holidays/GB.pm:# ABSTRACT: Determine British holidays - Current UK public and bank holiday dates up to 2019
META.json:   "abstract" : "Determine British holidays - Current UK public and bank holiday dates up to 2019",
META.yml:abstract: 'Determine British holidays - Current UK public and bank holiday dates up to 2019'

```

**Comment by mjemmeson** [comment]
Ah, thanks again! Looks like I need to examine my dzil config again, been a while since I created this repo

**Comment by mjemmeson** [comment]
Seems fixed everywhere now in V0.015. Thanks again!
